### ISSUE TITLE
Destructuring assignment not compatible with node 4

### ISSUE THREAD

**Issue by janriemer** [ranisalt/node-argon2#54]
_First of all, sorry for ignoring your ISSUE_TEMPLATE, but this issue doesn't need it._

According to the [es6 compat table](https://kangax.github.io/compat-table/es6/), Node v4 does **not support destructuring assignment**.

However, it is used [at line 20 of index.js](https://github.com/ranisalt/node-argon2/blob/master/index.js#L20) and therefore I run into a syntax error (full error see below).

When we really want to use the latest stuff, maybe we can introduce a postinstall script where babel transpiles the code for Node 4 compatibility. However, if this is the only line, it might not be worth it.

Thanks for your effort. :tulip: 

### Environment
**Operating system**: Linux

**Node version:** 4.5.0

### Error stack trace:
```
const {max, min} = limits[key]

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

**Comment by julien-f** [comment]
Same problem here.

It's a real issue because Node 4 is the current LTS :/

**Comment by ranisalt** [comment]
Yeah, just noticed that. I believe it didn't trigger on tests because babel was silently transpiling index.js incorrectly too. It's failing to build with error `undefined undefined undefined` so... yeah, nice error tracing 👍

**Comment by ranisalt** [comment]
Published v0.14 that fixes this issue.

**Comment by janriemer** [comment]
Nice, thanks! :+1:
